shipdata/ShipStack.cxx

```diff
@@ -199,7 +199,7 @@ TParticle* ShipStack::GetCurrentTrack() const
 {
   TParticle* currentPart = GetParticle(fCurrentTrack);
   if ( ! currentPart) {
-    LOG(WARNING) << "ShipStack: Current track not found in stack!";
+    LOG(WARNING << "ShipStack: Current track not found in stack!";
```

Comment:
Does this line miss a parenthesis?

Reply:
Yes. I guess the compiler didn't catch it because it's a macro...

Reply:
Addressed in #372